docs-site/package.json

```
@@ -38,11 +38,11 @@
 	},
 	"devDependencies": {
 		"docusaurus-plugin-typedoc": "^0.17.2",
-		"typedoc": "^0.22.10",
-		"typedoc-plugin-markdown": "^3.11.9",
-		"typedoc-plugin-mdn-links": "^1.0.4",
-		"typedoc-plugin-missing-exports": "^0.22.6",
-		"typedoc-plugin-rename-defaults": "^0.4.0",
-		"typescript": "4.6.3"
+		"typedoc": "^0.23.14",
+		"typedoc-plugin-markdown": "^3.13.6",
+		"typedoc-plugin-mdn-links": "^1.0.6",
+		"typedoc-plugin-missing-exports": "^0.23.0",
+		"typedoc-plugin-rename-defaults": "^0.6.4",
+		"typescript": "^4.8.3"
```

I presume there was a good reason for pinning the TypeScript version in the first place here?
```suggestion
		"typescript": "~4.8.3"
```